core/src/main/java/brooklyn/config/BrooklynServerConfig.java

```diff
@@ -171,15 +167,6 @@ public static File getBrooklynWebTmpDir(ManagementContext mgmt) {
     }
 
     /**
-     * @return the CAMP platform associated with a management context, if there is one.
-     */
-    public static Maybe<CampPlatform> getCampPlatform(ManagementContext mgmt) {
```

Comment:
All usages (together with the accessor) are moved to the `brooklyn-camp` project.